mpa/cls/explainer.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os.path as osp
+import torch
+
+import mmcv
+from mmcv.parallel import MMDataParallel
+from mmcv.runner import load_checkpoint, wrap_fp16_model
+
+from mmcls.datasets import build_dataloader, build_dataset
+from mmcls.models import build_classifier
+
+from mpa.registry import STAGES
+from mpa.cls.stage import ClsStage
+from mpa.modules.hooks.recording_forward_hooks import ActivationMapHook, EigenCamHook
+from mpa.utils.logger import get_logger
+logger = get_logger()
+EXPLAINER_HOOK_SELECTOR = {
+    'eigencam': EigenCamHook,
+    'activationmap': ActivationMapHook}
```

Comment:
Is this right format? You may want to run local pre-commit test if CI is not working well.

Reply:
Found inconsistency across this PR: 'str' or "str"?

Reply:
Yes, all changed to 'str', following other files and the `EXPLAINER_HOOK_SELECTOR` one does not occur error in pre-commit but modified to split lines for readability.